spring-boot-admin-server-ui/src/main/frontend/index.js

```diff
@@ -38,6 +41,8 @@ const installables = [
   ...global.SBA.extensions
 ];
 
+let vueI18Next = new VueI18Next(i18n);
+
 installables.forEach(view => view.install({
```

Comment:
Would it be somehow possible to pass along the 18n so that translations could be added in custom extensions?

Reply:
I have added an example to `sample-custom-ui` using i18n :)